go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/containerd/v2
 
-go 1.21
+go 1.22.0
```

Comment:
(Not sure whether this should be 1.22.2 or just 1.22.0)

Reply:
If we upgrade to 1.22.2, it will force anyone using containerd module to be upgraded to 1.22.2. I am good with keeping at 1.22.0